lib/rubber/cloud/aws/classic.rb

```diff
@@ -24,7 +24,7 @@ def describe_security_groups(group_name=nil)
 
         opts = {}
         opts["group-name"] = group_name if group_name
-        response = compute_provider.security_groups.all(opts)
+        response = compute_provider.security_groups.all(opts).reject { |group| group.vpc_id }
```

Comment:
Doesn't security_groups#all return something array-like? Won't this just return an empty list?

Reply:
So I can't retest that because we don't have Classic instances anymore but as far as I remember the problem was that it was returning both classic & VPC security groups (it's a bit ridiculous but you can have both named exactly that same!). With this change it should operate only on Classic security groups as it is expected.